python/svi_client/setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = hydrotools.svi_client
+version = attr: hydrotools.svi_client._version.__version__
+author = Austin Raney
+author_email = aaraney@protonmail.com
+description = Retrieve Social Vulnerability Index data from The Center for Disease Control / The Agency for Toxic Substances and Disease Registry.
+long_description = file: README.md
+long_description_content_type = text/markdown
+charset = UTF-8
+license = USDOC
+license_files =
+    LICENSE
+url = https://github.com/NOAA-OWP/hydrotools
+project_urls =
+    Documentation = https://noaa-owp.github.io/hydrotools/hydrotools.svi_client.html
+    Source = https://github.com/NOAA-OWP/hydrotools/tree/main/python/svi_client
+    Tracker = https://github.com/NOAA-OWP/hydrotools/issues
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    License :: Free To Use But Restricted
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering :: Hydrology
```

Comment:
This classifier should probably be removed.

Reply:
Agreed. Possible replacements:
`Topic :: Scientific/Engineering`
`Topic :: Sociology`

Possible addition:
`Intended Audience :: Science/Research`

Reply:
Updated to reflect @jarq6c's suggestions. Thanks!